Sources/Fuzzilli/FuzzIL/JsOperations.swift

```diff
@@ -1403,12 +1403,19 @@ public enum BinaryOperator: String, CaseIterable {
     case RShift   = ">>"
     case Exp      = "**"
     case UnRShift = ">>>"
+    // Nullish coalescing operator (??)
+    case NCO = "??"
```

Comment:
'NCO' is maybe a little weird because none of the other operators have "operator" in their name. 'NC' is maybe a bit short/unintuitive though, so how about just calling it `NullCoalesce`?

Reply:
Good point. Thanks!